components/service/glean/src/test/java/mozilla/components/service/glean/BooleanMetricTypeTest.kt

```diff
@@ -64,19 +65,14 @@ class BooleanMetricTypeTest {
 
         // Record two booleans of the same type, with a little delay.
         booleanMetric.set(true)
-
         // Check that data was properly recorded.
-        val snapshot = BooleansStorageEngine.getSnapshot(storeName = "store1", clearStore = false)
-        assertEquals(1, snapshot!!.size)
-        assertEquals(true, snapshot.containsKey("telemetry.boolean_metric"))
-        assertEquals(true, snapshot.get("telemetry.boolean_metric"))
+        assertTrue(booleanMetric.testHasValue())
```

Comment:
*NEAT*!

Reply:
Thanks for the suggestions, after this pass I'm MUCH happier with the test API functions and how they look and feel.